eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/jobs/steps/processors/SubSnpNoHgvsToVariantProcessorTest.java

```diff
@@ -0,0 +1,380 @@
+                                                     Orientation.FORWARD, Orientation.REVERSE, Orientation.FORWARD,
+                                                     CONTIG_START, false, true, "G", CREATED_DATE, TAXONOMY);
+
+        List<DbsnpSubmittedVariantEntity> variants = processor.process(subSnpNoHgvs);
+
+        assertProcessedVariant(subSnpNoHgvs, variants.get(0), CHROMOSOME, CHROMOSOME_START, "G", "A", true, true);
+    }
+
+    @Test
+    public void transformMultiallelicSS() throws Exception {
+        SubSnpNoHgvs subSnpNoHgvs = new SubSnpNoHgvs(1975823489L, 13637891L, "A/C/G", ASSEMBLY, BATCH_HANDLE,
+                                                     BATCH_NAME, CHROMOSOME, CHROMOSOME_START, CONTIG_NAME,
+                                                     DbsnpClass.SNV, Orientation.FORWARD, Orientation.FORWARD,
+                                                     Orientation.FORWARD, CONTIG_START, false, false, "A", CREATED_DATE,
+                                                     TAXONOMY);
+
+        List<DbsnpSubmittedVariantEntity> variants = processor.process(subSnpNoHgvs);
+
+        assertProcessedVariant(subSnpNoHgvs, variants.get(0), "A", "C");
+        assertProcessedVariant(subSnpNoHgvs, variants.get(1), "A", "G");
+    }
+
+    @Test
+    public void transformMNV() throws Exception {
+        SubSnpNoHgvs subSnpNoHgvs = new SubSnpNoHgvs(666079762L, 313826846L, "TTA/ATC", ASSEMBLY, BATCH_HANDLE,
+                                                     BATCH_NAME, CHROMOSOME, CHROMOSOME_START, CONTIG_NAME,
+                                                     DbsnpClass.MVN, Orientation.FORWARD, Orientation.REVERSE,
+                                                     Orientation.REVERSE, CONTIG_START, false, false, "TAA",
+                                                     CREATED_DATE, TAXONOMY);
+
+        List<DbsnpSubmittedVariantEntity> variants = processor.process(subSnpNoHgvs);
+
+        assertProcessedVariant(subSnpNoHgvs, variants.get(0), "TTA", "ATC");
+    }
+
+    @Test
+    public void transformMultiallelicMNV() throws Exception {
+        SubSnpNoHgvs subSnpNoHgvs = new SubSnpNoHgvs(973513424L, 525103154L, "AC/TC/TA", ASSEMBLY, BATCH_HANDLE,
+                                                     BATCH_NAME, CHROMOSOME, CHROMOSOME_START, CONTIG_NAME,
+                                                     DbsnpClass.MVN, Orientation.FORWARD, Orientation.FORWARD,
+                                                     Orientation.FORWARD, CONTIG_START, false, false, "AC",
+                                                     CREATED_DATE, TAXONOMY);
+
+        List<DbsnpSubmittedVariantEntity> variants = processor.process(subSnpNoHgvs);
+
+        assertProcessedVariant(subSnpNoHgvs, variants.get(0), "AC", "TC");
+        assertProcessedVariant(subSnpNoHgvs, variants.get(1), "AC", "TA");
+    }
+
```

Comment:
Another test for INDEL (something like C/TA)? Must not be technically different from MNV but it would be good to have tests with alternates having unequal lengths.

Reply:
🎉